src/main/java/model/Crew.java

```diff
@@ -0,0 +1,41 @@
+package model;
+
+import java.util.Objects;
+
+public class Crew {
+
+    private final String nickname;
+
+    private Crew(String nickname) {
+        this.nickname = nickname;
+    }
+
+    public static Crew of(String nickname) {
+        return new Crew(nickname);
+    }
+
+    public boolean isEqualName(String nickname) {
+        return nickname.equals(nickname);
+    }
```

Comment:
이 메서드 때문인 것 같은데요. 테스트가 존재하지 않네요. ㅠㅠ 전체적으로 테스트는 확인이 필요합니다. 
```suggestion
    public boolean isEqualName(String nickname) {
        return this.nickname.equals(nickname);
    }
```

Reply:
아..ㅠㅠ 그러네요! 진행하면서 무작정 만든 메서드로 테스트 없이 진행했던 것 같아요. 시간이 부족해서 급하게 구현하다 보니 발생한 문제인 것 같네요. 😢 감사합니다!